iscsi/iscsi.go

```diff
@@ -239,13 +231,17 @@ func getMultipathDevice(devices []Device) (*Device, error) {
 	return multipathDevice, nil
 }
 
-// Connect is for backward-compatiblity with c.Connect()
-func Connect(c Connector) (string, error) {
-	return c.Connect()
+// Connect is for backward-compatibility with c.Connect()
```

Comment:
Is this still needed? Adding ctx must break whatever backward compatibility this function used to provide.